source/nodejs/tests/unit-tests/src/unit-tests.test.ts

```diff
@@ -1,4 +1,7 @@
 import * as AdaptiveCards from "adaptivecards";
+import * as ACData from "adaptivecards-templating";
+
+var fs = require("fs");
```

Comment:
not sure if the `unit-tests` project is set up correctly, but you may be able to do:

```suggestion
import { readFileSync } from "fs";
```

(and then refer to it as just `readFileSync` in `loadFile()` below)

if it doesn't work, feel free to leave as-is :)